src/main/java/binnie/core/machines/RendererMachine.java

```diff
@@ -9,7 +9,7 @@
 import net.minecraft.world.IBlockAccess;
 import net.minecraft.world.World;
 
-// TODO unused class?
+// Hmm. It's used class
```

Comment:
IDEA says that class is unused. But if drop it, genetic machines aren't render o_O
Many strange code...